apps/verify/models/read-cookie.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const dataService = require('../../common/data-service');
+const {
+  makeRequest,
+  requestHeaders,
+} = dataService;
+
+/**
+ * API get - check nrm cookie
+ *
+ * @param {string} uuid - unique id for the cookie
+ *
+ * @returns {Promise} - response from data service (the row data)
+ */
+const readDataFromStore = async(uuid) => {
+
+  const options = {
+    method: 'get',
+    body: null,
+    headers: requestHeaders,
```

Comment:
As `headers` is a common element in all of these, and are just coming straight from `../../common/data-service`, would it be better to just bake the headers into `makeRequest()` itself?

Reply:
copy pasted from Julian tbh

Reply:
I'll fix or @jjrobothamho will fix in another PR